controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook.go

```diff
@@ -105,6 +105,7 @@ func (in *KubeadmControlPlane) ValidateUpdate(old runtime.Object) error {
 		{spec, kubeadmConfigSpec, preKubeadmCommands},
 		{spec, kubeadmConfigSpec, postKubeadmCommands},
 		{spec, kubeadmConfigSpec, files},
+		{spec, kubeadmConfigSpec, "verbosity"},
```

Comment:
this should be defined as a const above like the others

Reply:
🤷 Some of these here are defined as consts and some others aren't

Reply:
ok, fair

these constants are that useful anyways especially when only used once 🤔